scripts/globals/job_utils/rune_fencer.lua

```diff
@@ -654,3 +683,47 @@ xi.job_utils.rune_fencer.usePflug = function(player, target, ability, action)
 
     player:addStatusEffect(xi.effect.PFLUG, baseStrength, 0, 120, 0, meritBonus)
 end
+
+-- see https://www.bg-wiki.com/ffxi/Gambit
+xi.job_utils.rune_fencer.useGambit = function(player, target, ability, action)
+    local highestRune = player:getHighestRuneEffect()
+    local weaponSkillType = player:getWeaponSkillType(xi.slot.MAIN)
+    local effects = player:getStatusEffects()
+    local SDTPower = -10
+    local jobPointBonusDuration = player:getJobPointLevel(xi.jp.GAMBIT_DURATION)
+    local gearBonusDuration = player:getMod(xi.mod.GAMBIT_DURATION)
+
+    action:speceffect(target:getID(), getSpecEffectElementEffusion(highestRune)) -- set element color for animation.
+    action:setAnimation(target:getID(), getAnimationEffusion(weaponSkillType, 10)) -- set animation for currently equipped weapon
+
+    SDTPower = SDTPower * 100 -- adjust to SDT modifier
+
+    local SDTTypes = {} -- one SDT type per rune which can be additive
+    local i = 0
+
+    for _, effect in ipairs(effects) do
```

Comment:
Do we not have a binding for getrunes? or similar?

Reply:
Not yet, but I can add one. I'll do that I get home.